docs/04-providers/07-mend-io.md

```
@@ -0,0 +1,17 @@
+---
+description: Renovate dependency updates
+---
+
+## Overview
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)